src/app/archive/create_schema.sql

```diff
@@ -19,92 +15,84 @@ CREATE TABLE blocks (
   ledger_proof_nonce int NOT NULL,
   status int NOT NULL,
   block_length int NOT NULL,
-  block_time timestamp NOT NULL
+  block_time bit(64) NOT NULL
```

Comment:
Why is this not a timestamp?

Reply:
I talked to @nholland94 and he suggested that we should use some long number, like bits, rather than timestamp. There could be a lot of confusion using timestamp.

Reply:
I'd be curious to hear more about this tradeoff at some point -- in my experience most of the reason that one might not want to use native timestamp types is related to needing to maintain compatibility across different flavours of sql.
If anything I would have asked why we weren't using [timestampz](https://wiki.postgresql.org/wiki/Don't_Do_This#Date.2FTime_storage) (with timezone)